src/iiif/utils_azure.py

```diff
@@ -65,6 +65,29 @@ def store_object_on_storage_account(local_zip_file_path, filename):
     return blob_client, blob_service_client
 
 
+def store_blob_on_storage_account(storage_container, blob_name, blob):
+    blob_client, blob_service_client = get_blob_client(
+        container_name=storage_container, blob_name=blob_name
+    )
+    blob_client.upload_blob(blob, overwrite=True)
+    return blob_client, blob_service_client
+
+
+def get_blob_from_storage_account(storage_container, blob_name):
+    blob_client, _ = get_blob_client(
+        container_name=storage_container, blob_name=blob_name
+    )
+    blob = blob_client.download_blob().readall()
+    return blob_client, blob
+
+
+def remove_blob_from_storage_account(storage_container, blob_name):
```

Comment:
Als de blobs een automatische ttl of expiry hebben dan is deze functie trouwens uberhaupt niet meer nodig.

Reply:
Vermoed van wel. Minder gebruik van storage. Dan kunnen we de ttl wat langer zetten. In dat geval kunnen we een cronjob hebben die het eventueel nogmaals probeert. Of als we een bug vinden kunnen de resterende nog worden verwerkt na een fix